packages/@sanity/default-layout/src/DefaultLayout.tsx

```diff
@@ -138,6 +138,7 @@ class DefaultLayout extends React.PureComponent<Props, State> {
     const {tools, router} = this.props
     const {createMenuIsOpen, menuIsOpen, searchIsOpen} = this.state
     const isOverlayVisible = menuIsOpen || searchIsOpen
+    const tool = router.state && router.state.tool ? router.state.tool : ''
```

Comment:
Nitpick: this could be simplified to `const tool = router.state?.tool`?

Is the idea of using the empty string (and not e.g. `undefined`) to not change the typing?

Reply:
👍
Also, since the majority of components which took the tool as input required a string, I felt it was a safer to let the default value be an empty string.